### PR TITLE
use alpine linux as base image

### PR DESCRIPTION
**Pull request by PascalBourdier**
centos 8 is EOL since 31/12/2021

### CODE REVIEW

**Review by grandria — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -13,10 +13,10 @@ jobs:
     name: lint
     runs-on: ubuntu-latest
     steps:
-      - name: Set up Go 1.15
+      - name: Set up Go 1.16
         uses: actions/setup-go@v2
         with:
-          go-version: 1.15
+          go-version: 1.16
```
Comment (tbeaugrand) [inline .github/workflows/ci.yml:19]:
```suggestion
          go-version: 1.17
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -28,10 +28,10 @@ jobs:
     name: test
     runs-on: ubuntu-latest
     steps:
-      - name: Set up Go 1.15
+      - name: Set up Go 1.16
         uses: actions/setup-go@v2
         with:
-          go-version: 1.15
+          go-version: 1.16
```
Comment (tbeaugrand) [inline .github/workflows/ci.yml:34]: same

File: Dockerfile [hunk 3]
```diff
@@ -1,10 +1,8 @@
-FROM golang:1.15-buster as app-builder
+FROM golang:1.16-alpine as app-builder
```
Comment (tbeaugrand) [inline Dockerfile:1]: same

File: Dockerfile [hunk 4]
```diff
@@ -13,17 +11,21 @@ RUN go mod download
 
 COPY . ./
 
-RUN make 
+RUN apk add build-base curl && make
+
 
-FROM golang:1.15-buster as lib-builder
+FROM golang:1.16-alpine as lib-builder
```
Comment (tbeaugrand) [inline Dockerfile:17]: 😗

**Review by atelal — APPROVED** [review]

**Pull request merged**